lib/matplotlib/afm.py

```diff
@@ -244,6 +244,8 @@ def _parse_char_metrics(fh):
         # Reference).
         if name == 'Euro':
             num = 128
+        if name == 'minus':
```

Comment:
```suggestion
        elif name == 'minus':
```
Just to be a bit more explicit.

Reply:
sure, done